quakemigrate/core/lib.py

```diff
@@ -93,8 +95,11 @@ def migrate(
 
     *grid_dimensions, n_luts = traveltimes.shape
     n_onsets, t_samples = onsets.shape
+    logging.debug((n_onsets, t_samples))
     n_samples = t_samples - first_idx - last_idx
+    logging.debug(n_samples)
     map4d = np.zeros(tuple(grid_dimensions) + (n_samples,), dtype=np.double)
+    logging.debug(map4d.shape)
```

Comment:
I think this is probably fine, but we might consider adding some information making clear what these numbers are when they are printed in the debug log. I can think of some scenarios where this would be useful, so think would just be more complete to have:

```suggestion
    logging.debug(f"(n_onsets, t_samples) : ({n_onsets}, {t_samples})")
    n_samples = t_samples - first_idx - last_idx
    logging.debug(f"n_samples : {n_samples}")
    map4d = np.zeros(tuple(grid_dimensions) + (n_samples,), dtype=np.double)
    logging.debug(f"map4d shape : {map4d.shape}")
```